### ISSUE TITLE
importerror: cannot import name '_assert_no_grad' 

### ISSUE THREAD

**Issue by liruolixi** [pytorch/pytorch#10989]
## Issue description
I run this code on linux. version:python3.5. 

when using that code 

from torch.nn.modules.loss import _assert_no_grad

I got that error : importerror: cannot import name '_assert_no_grad'

**Comment by zou3519** [comment]
I believe we removed _assert_no_grad. If you're running a user-defined script you should correct that script to match.
